eeg_flow/tasks/epochs_evoked.py

```diff
@@ -0,0 +1,434 @@
+        events=events,
+        event_id=events_id,
+        tmin=metadata_tmin,
+        tmax=metadata_tmax,
+        sfreq=raw.info["sfreq"],
+        row_events=row_events,
+    )
+
+    conditions = [
+        metadata["event_name"].eq("target") & pd.notna(metadata["response"]),
+        metadata["event_name"].eq("target") & pd.isna(metadata["response"]),
+        metadata["event_name"].eq("standard") & pd.notna(metadata["response"]),
+        metadata["event_name"].eq("standard") & pd.isna(metadata["response"]),
+        metadata["event_name"].eq("novel") & pd.notna(metadata["response"]),
+        metadata["event_name"].eq("novel") & pd.isna(metadata["response"]),
+    ]
+    choices = [
+        "Hits",
+        "Misses",
+        "FalseAlarms",
+        "CorrectRejections",
+        "FalseAlarms",
+        "CorrectRejections",
+    ]
+
+    metadata["response_type"] = np.select(conditions, choices, default=0)
+    metadata["response_type"].value_counts()
+    
+    ########
+    metadata.response_correct = False
+    metadata.loc[
+        (metadata["response_type"] == "CorrectRejections"), "response_correct"
+    ] = True
+    metadata.loc[(metadata["response_type"] == "Hits"), "response_correct"] = True
+
+    metadata.loc[
+        (metadata["response_type"] == "FalseAlarms"), "response_correct"
+    ] = False
+    metadata.loc[(metadata["response_type"] == "Misses"), "response_correct"] = False
+
+
+    return metadata, events, event_id
+
+def get_SDT_outcomes(metadata):
+    
+    num_Hits = len(metadata[metadata["response_type"] == "Hits"])
+    num_CorrectRejections = len(
+        metadata[metadata["response_type"] == "CorrectRejections"]
+    )
+    num_Misses = len(metadata[metadata["response_type"] == "Misses"])
+    num_FalseAlarms = len(metadata[metadata["response_type"] == "FalseAlarms"])
+
+    return(num_Hits, num_CorrectRejections, num_Misses, num_FalseAlarms)
+
+def plot_RT(hits, fname_stem, derivatives_subfolder, response_mean, response_std):
+    # visualize response times of TP
+    ax_rt = hits["response"].plot.hist(
+        bins=100,
+        title=f"Response Times of TPs\nmean:{str(response_mean)} ({str(response_std)})",
+    )
+
+    fname_rt_plot = derivatives_subfolder / "plots" / (fname_stem + "_step9_RT.svg")
+    ax_rt.figure.suptitle(fname_stem, fontsize=16, y=1)
+    ax_rt.figure.savefig(fname_rt_plot, transparent=True)
+    ax_rt
+
+def get_indiv_behav(metadata, num_Hits, num_CorrectRejections, num_Misses, num_FalseAlarms, 
+                    fname_stem, derivatives_subfolder, response_mean, response_std):
+    correct_response_count = metadata["response_correct"].sum()
+
+    print(
+        f"\nCorrect responses: {correct_response_count}\n"
+        f"Incorrect responses: {len(metadata) - correct_response_count}\n"
+    )
+
+    print("Hits, Misses, Correct Rejections, False Alarms")
+    print(num_Hits, num_Misses, num_CorrectRejections, num_FalseAlarms,"\n")
```

Comment:
Don't print, log. A logger is better because `print` only outputs to `sys.stdout`, while a log has:
- different levels to filter the importance of a message
- filters
- different handlers to output to `sys.stdout`, and to other I/O streams.

For instance, if your code has some nice logging, you can add a file handler which will automatically save all logs of a given level (and above) to disk. And that level can be different from the level of a second handler outputting in `sys.stdout`, as print would.

`eeg-flow`  has a logger defined at the root of the package:

```
from eeg_flow import add_file_handler, logger, set_log_level

set_log_level("INFO")
add_file_handler("my-logs.log", verbose="DEBUG")
logger.info("test 1")  # outputs both in the console and in the file `my-logs.log`
logger.debug("debug line")  # outputs only in the file `my-logs.log`)
```